packages/host/tests/acceptance/code-submode-test.ts

```diff
@@ -999,7 +999,7 @@ module('Acceptance | code submode tests', function (hooks) {
     });
   });
 
-  test('has a profile icon in the bottom left corner', async function (assert) {
+  test('has a profile icon in the bottom left corner that opens profile info popover', async function (assert) {
```

Comment:
Maybe this belongs more in `operator-mode-test`? Since it shows in both submodes.

Reply:
Good idea